tutor/src/components/my-courses/listings.jsx

```diff
@@ -196,7 +196,7 @@ function ExploreAPreview({ course }) {
 }
 
 @observer
-export class MyCoursesPreview extends React.PureComponent {
+export class MyCoursesPreview extends React.Component {
 
```

Comment:
the change here is that we're no longer using PureComponent because it'll cause an error when/if we ever upgrade react